src/app/components/harvest/screens/metadata-review/metadata-review.component.ts

```diff
@@ -285,7 +294,7 @@ export class MetadataReviewComponent
     );
     this.tableLoading = false;
 
-    if (harvestItems.length === 0) {
+    if (harvestItems === undefined) {
```

Comment:
this seems like an incorrect change - the api should never be returning undefined for harvest items and now the condition no longer works for arrays of size 0. why was this changed?

Reply:
I've reverted this change. It's an artifact from my testing